File: lab_2.py
# 211RDB105 Artjoms Kučerjavijs

def input_sequence(list):
    x = int(input())
    while x != 0:
        list.append(x)
        x = int(input())

first, second, res = [], [], []
input_sequence(first)
input_sequence(second)
for first_el in first:
    if not first_el in second:
        res.append(first_el)
for num in res:
    print(num)

File: lab_2.cpp
// 211RDB105 Artjoms Kučerjavijs
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;


void inputSequence(vector<int>& array){
    int input = 1;
    bool flag = false;
    while (input != 0){
        scanf("%d", &input);
        array.push_back(input);
    }
}

int main() {
    vector<int> first, second;
    inputSequence(first);
    inputSequence(second);
    for(int i = 0; i < first.size(); i++){
        if (count(begin(second), end(second), first[i]) < 1 ){
            cout<<first[i]<<endl;
        }
    }
    return 0;
}